mkdocs.yml

```diff
@@ -19,9 +19,9 @@ pages:
           - 'HA': 'data-management/cygnus/ha/readme.md'
         - 'QuantumLeap':
           - 'Intro': 'data-management/quantumleap/readme.md'
-    - 'IoT Services Enablement': 
+    - 'IoT Services Enablement':
         - 'IDAS': 'iot-services/readme.md'
-#        - 'IoT Agent JSON': 'iot-services/iotagent-json/readme.md'
+        - 'IoT Agent JSON': 'iot-services/iotagent-json/README.md'
```

Comment:
@chicco785 this should do. I run `mkdocs serve` locally from time to time to read docs I'm currently working on. However, this may have escaped my mkdoc's checker since mac's FS is case insensitive so the file was actually found in the mac, but then of course not in the [job running in linux](https://travis-ci.org/smartsdk/smartsdk-recipes/jobs/308918164).